evolver/settings.py

```diff
@@ -13,6 +15,13 @@ class Settings(BaseSettings):
     DEFAULT_NUMBER_OF_VIALS_PER_BOX: int = 16
     LOG_LEVEL: str = "INFO"
     LOG_FORMAT: str = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+    ROOT_CALIBRATOR_FILE_STORAGE_PATH: Path = Path("calibration_files")
+    DEFAULT_CALIBRATION_EXPIRE: timedelta = Field(
+        timedelta(weeks=4 * 6),  # Approx. 6 months.
```

Comment:
Should we make this ``None``?

Reply:
Also, arguably this should use ``evolver.base.ExpireField``, however, that would result in circular imports as soon as ``evolver.base`` imported settings (which is likely).

Reply:
Could move the dec to here instead.